ui/components/app/transaction-breakdown/transaction-breakdown.component.js

```diff
@@ -85,19 +91,71 @@ export default class TransactionBreakdown extends PureComponent {
             />
           </TransactionBreakdownRow>
         )}
-        <TransactionBreakdownRow title={t('gasPrice')}>
-          {typeof gasPrice === 'undefined' ? (
-            '?'
-          ) : (
-            <CurrencyDisplay
-              className="transaction-breakdown__value"
-              data-testid="transaction-breakdown__gas-price"
-              currency={nativeCurrency}
-              denomination={GWEI}
-              value={gasPrice}
-              hideLabel
+        {process.env.SHOW_EIP_1559_UI && (
+          <TransactionBreakdownRow title={t('transactionHistoryBaseFee')}>
+            {typeof baseFee === 'undefined' ? (
```

Comment:
Just for my learning purposes, why can we get 'undefined' as a string inside `baseFee`?

Reply:
its `typeof baseFee` which will be `'undefined'` if the value of baseFee is `undefined`

Reply:
Ha, typeof is there, never-mind :)